MapleServer2/Types/Currency.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MapleServer2.Enums;
+
+namespace MapleServer2.Types
+{
+    public class Currency
+    {
+
+        public long Meso { get; set; }
+        public long Meret { get; set; }
+        public long ValorToken { get; set; }
+        public long Treva { get; set; }
+        public long Rue { get; set; }
+        public long HaviFruit { get; set; }
+        public long MesoToken { get; set; }
+
+        public Currency()
```

Comment:
I think the idea for this class was to have each individual currency represented by this single class, rather than one class representing them all. This class would also handle manipulating the amount, minus the packet sending, which can still be done by the `Player.cs` class